setup/symfony_server.rst

```diff
@@ -0,0 +1,294 @@
+
+This command creates a local certificate authority, registers it in your system
+trust store, registers it in Firefox (this is required only for that browser)
+and creates a default certificate for ``localhost`` and ``127.0.0.1``. In other
+words, it does everything for you.
+
+Before browsing your local application with HTTPS instead of HTTP, restart its
+server stopping and starting it again.
+
+Different PHP Settings Per Project
+----------------------------------
+
+Selecting a Different PHP Version
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have multiple PHP versions installed on your computer, you can tell
+Symfony which one to use creating a file called ``.php-version`` at the project
+root directory:
+
+.. code-block:: terminal
+
+    $ cd my-project/
+
+    # use a specific PHP version
+    $ echo "7.2" > .php-version
+
+    # use any PHP 7.x version available
+    $ echo "7" > .php-version
+
+.. tip::
+
+    The Symfony server traverses the directory structure up to the root
+    directory, so you can create a ``.php-version`` file in some parent
+    directory to set the same PHP version for a group of projects under that
+    directory.
+
+Run command if you don't remember all the PHP versions installed on your
+computer:
+
+.. code-block:: terminal
+
+    $ symfony local:php:list
+
+      # You'll see all supported SAPIs (CGI, FastCGI, etc.) for each version.
+      # FastCGI (php-fpm) is used when possible; then CGI (which acts as a FastCGI
+      # server as well), and finally, the server falls back to plain CGI.
+
+Overriding PHP Config Options Per Project
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can change the value of any PHP runtime config option per project by creating a
+file called ``php.ini`` at the project root directory. Add only the options you want
+to override:
+
+.. code-block:: terminal
+
+    $ cd my-project/
+
+    # this project only overrides the default PHP timezone
+    $ cat php.ini
+    [Date]
+    date.timezone = Asia/Tokyo
+
+Running Commands with Different PHP Versions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When running different PHP versions it's useful to use the main ``symfony``
+command as a wrapper for the ``php`` command. This allows you to always select
+the most appropriate PHP version according to the project which is running the
+commands. It also loads the env vars automatically, which is important when
+running non-Symfony commands:
+
+.. code-block:: terminal
+
+    # runs the command with the default PHP version
+    $ php -r "..."
+
+    # runs the command with the PHP version selected by the project
+    # (or the default PHP version if the project didn't select one)
+    $ symfony php -r "..."
+
+If you are using this wrapper frequently, consider aliasing the ``php`` command
+to it:
+
+.. code-block:: terminal
+
+    $ cd ~/.symfony/bin
+    $ cp symfony php
+    # now you can run "php ..." and the "symfony" command will be executed instead
```

Comment:
What about adding a note about the other wrappers that could be useful (`php-config`, `pear`, `pecl`, ...)?